### PR TITLE
TxtSync SMS Connector - ready for certification

### PR DESCRIPTION
**Pull request by TxtSync**
---
When submitting a connector, please check the following conditions for your PR.

- [X] I attest that the connector works and I verified by deploying and testing all the operations.
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a logo.

### CODE REVIEW

File: certified-connectors/TxtSync/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2235 @@
...
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "FromNumber": {
+                    "type": "string",
+                    "description": "The Sender ID or E164 number to send from",
+                    "title": "From Number",
+                    "x-ms-visibility": ""
```
Comment (sriyen-msft) [inline certified-connectors/TxtSync/apiDefinition.swagger.json:53]: @TxtSync Can you please let us know why `x-ms-visibility` is set to empty string? If you do not want it to add, you can remove it.

Comment (sriyen-msft) [reply certified-connectors/TxtSync/apiDefinition.swagger.json:53]: Applies this comment the similar properties in the swagger

File: certified-connectors/TxtSync/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2235 @@
...
+        "summary": "Send an SMS",
+        "description": "Sends out a single SMS to an E164 number",
+        "operationId": "SendSMS",
+        "parameters": [
+          {
+            "name": "x-api-key",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "M3Yf9L0njL8eL9ukphahq9zk5LFRIOvn2fT4BcmV",
```
Comment (sriyen-msft) [inline certified-connectors/TxtSync/apiDefinition.swagger.json:109]: @TxtSync We are receiving security alerts regarding secret-look-a-like values. Please remove them from the swagger.

Comment (sriyen-msft) [reply certified-connectors/TxtSync/apiDefinition.swagger.json:109]: Applies this comment to similar properties and values in the swagger

File: certified-connectors/TxtSync/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2235 @@
...
+                "From",
+                "Message"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/system/applications/2/webhooks": {
+      "x-ms-notification-content": {}
```
Comment (sriyen-msft) [inline certified-connectors/TxtSync/apiDefinition.swagger.json:289]: @TxtSync Empty operations can be removed

File: certified-connectors/TxtSync/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,2235 @@
...
+                  },
+                  "AddressLine2": {
+                    "type": "string",
+                    "title": "Address Line2",
+                    "description": "Address line"
+                  },
+                  "City": {
+                    "type": "string",
+                    "title": "City",
+                    "description": "Address line"
```
Comment (sriyen-msft) [inline certified-connectors/TxtSync/apiDefinition.swagger.json:1955]: @TxtSync Typo? It might be `"City"`

Comment (sriyen-msft) [reply certified-connectors/TxtSync/apiDefinition.swagger.json:1955]: Applies this comment to similar properties below

File: certified-connectors/TxtSync/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,2235 @@
...
+                  "LastCommunicationDate": {
+                    "type": "string",
+                    "description": "The date (UTC) that the contact last communicated",
+                    "title": "Last Communication Date",
+                    "format": "date-time"
+                  },
+                  "AllowSMS": {
+                    "type": "boolean",
+                    "description": "Whether the contact wishes to receive SMS",
+                    "title": "AllowSMS"
```
Comment (sriyen-msft) [inline certified-connectors/TxtSync/apiDefinition.swagger.json:1981]: @TxtSync There are few title values are looking like operation Ids. We expect title values must be free from grammatical and spelling mistakes.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Review by mamurshe — DISMISSED** [review]
Please use the set header policy instead of using x-api-key for every operation https://docs.microsoft.com/en-us/connectors/custom-connectors/policy-templates/setheader/setheader

**Review by TxtSync — COMMENTED** [review]
As notified to the PowerApps Team via Email

**Review by TxtSync — COMMENTED** [review]
All changes made as requested

File: certified-connectors/TxtSync/apiDefinition.swagger.json (hunk 1 above)
Comment (sriyen-msft) [reply certified-connectors/TxtSync/apiDefinition.swagger.json:53]: @TxtSync Looks like it is not fixed, empty values for x-ms-visibility are not allowed. If you have ay specific reason to keep empty strings, please let us know.

File: certified-connectors/TxtSync/apiDefinition.swagger.json (hunk 3 above)
Comment (sriyen-msft) [reply certified-connectors/TxtSync/apiDefinition.swagger.json:278]: @TxtSync Please remove this block, Please let us know the reason if you want to keep this.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by TxtSync** [comment]
All addressed on latest commit

**Comment by TxtSync** [comment]
Can this get a review please

File: certified-connectors/TxtSync/apiDefinition.swagger.json (hunk 2 above)
Comment (sriyen-msft) [reply certified-connectors/TxtSync/apiDefinition.swagger.json:102]: @TxtSync This is not resolved, please remove the property from the swagger.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**